proto/gds/members/v1alpha1/members.proto

```diff
@@ -70,4 +73,7 @@ message SummaryReply {
     int32 vasps = 1;
     int32 certificates_issued = 2;
     int32 new_members = 3;
+
+    // Details for the requested VASP
+    VASPMember vasp = 4;
```

Comment:
Same as above, should we change this to `member_info` or something similar? Does that make it more clear or does it just make "member" a confusing synonym with `vasp`?

Reply:
I agree that's a better name since we're in the members definitions. I think we've been using them interchangeably in a few places but here it does make a lot of sense to distinguish from the `vasps` parameter.